Cargo.toml

```diff
@@ -7,6 +7,10 @@ members = [
   "glean-core/benchmark",
 ]
 
+exclude = [
```

Comment:
OH! TIL!

Does it make sense to exclude the tests as well, so that they don't get vendored, for example, in m-c? (not for this PR)

Reply:
This line here only excludes it from the workspace. Otherwise `cargo build` won't work inside the sample directory.
Reason I did not put it in the workspace was that it should be more of a standalone example (but then again I think initially there was another blocker which I got rid off and now it can be reasonably part of the workspace without real issues, then it would share build artifacts). I'll check that again.

Re bundled tests: One _can_ exclude that in the respective `Cargo.toml` of a crate (`include` and `exclude` lists), but there's one advantage of bundling them:
The Rust project regurlarly runs ALL tests of ALL crates on crates.io to find potential breakage in the compiler. It'd be good if our crates are part of that.

Reply:
> The Rust project regurlarly runs ALL tests of ALL crates on crates.io to find potential breakage in the compiler. It'd be good if our crates are part of that.

TIL2 :)

Reply:
Gonna leave that as is:

We need to run with `lto` enabled, but these need to be configured for the whole workspace.
We could run the multi-bindings example in release mode, but that takes longer and I don't think it's worth it.
The majority of people won't run that example anyway, so the additional compilation and storage overhead is negligible for devs.